mlflow/langchain/__init__.py

```diff
@@ -270,6 +273,19 @@ def load_retriever(persist_directory):
         _MODEL_TYPE_KEY: lc_model.__class__.__name__,
         **model_data_kwargs,
     }
+
```

Comment:
@liangz1 We also need to identify any generative models, e.g. `langchain.llms.Databricks` or `DatabricksChat` that exist in the chain and extract the endpoint names

Reply:
Supported!